src/app/healthyrelationships/healthyrelationships.page.html

```diff
@@ -23,137 +23,134 @@
 
   <div *ngIf="subTab==='healthy'">
     <h1 style="font-size: 18px;">
-      Healthy, whether it's friendship or romantic
+      Signs of a Healthy Relationship
     </h1>
+    <h3 style="text-align: center; font-size: 14px;">
+      (friendship or romantic)
```

Comment:
Add added this sub-header to makes sure the distinction is still there, but potentially we remove this for consistency with the other sub-tabs